mon-pix/app/adapters/application.js

```diff
@@ -33,11 +33,11 @@ export default class Application extends JSONAPIAdapter.extend(DataAdapterMixin)
   }
 
   get _locale() {
-    if (this.intl.get('locale') === 'fr') {
+    if (this.intl.get('locale')[0] === 'fr') {
```

Comment:
on a la garantie que `this.intl.get('locale')` renvoie toujours une collection ? (et pas undefined parfois ?)

Reply:
et pourquoi ça marchait avant, sans chercher le [0] ?

Reply:
On lui défini dans la route application donc pas de risque d'undefined :  
https://github.com/1024pix/pix/blob/31342431f3255a1288fef5d12c2e03158923fcf1/mon-pix/app/routes/application.js#L47-L52

Reply:
ça a jamais marché ;)